compose.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  resourcematch-app:
+    build:
+      context: ./resourcematch-app
+    volumes:
+      - ./resourcematch-app:/code
+      - /code/node_modules
```

Comment:
It's mostly a developer convenience so, when bouncing between repos, we can do a quick `nvm use/install` to make sure we are using the appropriate version of `node` for that codebase. Since we update the version of `node` in our (maintained) repos when we do quarterly updates, any drift probably won't break us. But, for example, if we stop maintaining things in a repo like `raise-management-app`, it's nice to be able to use `nvm` vs trying to figure out what version of `node` is compatible, etc.

Reply:
Ahhhhh that makes sense. We aren't restricted to using the node version in the `.nvmrc`, but that node version tells us what version of `node` should be compatible with the project.